res/lang/news/news_de.txt

```diff
@@ -114,52 +114,52 @@ SCORE_POINTS = Punkte
 SPORT_ICEHOCKEY = Eishockey
 SPORT_SOCCER    = Fussball
 
-SPORT_SOCCER_TEAMREPORT_MATCHWIN_S1 = und Stürmer %TEAM1STAR% gewinnt %MATCHKIND% gegen
-SPORT_SOCCER_TEAMREPORT_MATCHWIN_P1 = und Stürmer %TEAM1STAR% gewinnen %MATCHKIND% gegen
-SPORT_SOCCER_TEAMREPORT_MATCHWIN_S2 = schlägt dank verwandelter Ecke durch %TEAM1STARSHORT% %MATCHKIND%
-SPORT_SOCCER_TEAMREPORT_MATCHWIN_P2 = schlagen dank verwandelter Ecke durch %TEAM1STARSHORT% %MATCHKIND%
-SPORT_SOCCER_TEAMREPORT_MATCHWIN_S3 = besiegt dank gehaltenem Elfmeter von Torwart %TEAM1KEEPERSHORT% %MATCHKIND%
-SPORT_SOCCER_TEAMREPORT_MATCHWIN_P3 = schlagen dank gehaltenem Elfmeter von Torwart %TEAM1KEEPERSHORT% %MATCHKIND%
-SPORT_SOCCER_TEAMREPORT_MATCHWIN_S4 = schlägt dank genialer Paraden von Torwart %TEAM1KEEPERSHORT% %MATCHKIND%
-SPORT_SOCCER_TEAMREPORT_MATCHWIN_P4 = schlagen dank genialer Paraden von Torwart %TEAM1KEEPERSHORT% %MATCHKIND%
-
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_S1 = gibt %MATCHKIND% 3 wertvolle Punkte an
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P1 = geben %MATCHKIND% 3 wertvolle Punkte an
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_S2 = unterliegt durch Schusselfehler von Torwart %TEAM1KEEPERSHORT% und %MATCHKIND% gegen
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P2 = unterliegen durch Schusselfehler von Torwart %TEAM1KEEPERSHORT% und %MATCHKIND% gegen
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_S3 = verliert mit enttäuschtem Torwart %TEAM1KEEPER% %MATCHKIND% gegen
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P3 = verlieren mit enttäuschtem Torwart %TEAM1KEEPER% %MATCHKIND% gegen
-
-SPORT_SOCCER_TEAMREPORT_MATCHKIND1 = nach %PLAYTIMEMINUTES% Minuten zweifelhaften Fussballs
-SPORT_SOCCER_TEAMREPORT_MATCHKIND2 = nach %PLAYTIMEMINUTES% Min taktischer Zweikämpfe
-SPORT_SOCCER_TEAMREPORT_MATCHKIND3 = nach spannenden %PLAYTIMEMINUTES% Minuten Rasensport
-
-SPORT_TEAMREPORT_MATCHWIN_S1 = besiegt dank %TEAM1STAR%
-SPORT_TEAMREPORT_MATCHWIN_P1 = besiegen dank %TEAM1STAR%
-SPORT_TEAMREPORT_MATCHWIN_S2 = schlägt %MATCHKIND%
-SPORT_TEAMREPORT_MATCHWIN_P2 = schlagen %MATCHKIND%
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_S1 = und Stürmer ${.self:"sportteam":0:"member":"forward":"fullname"} gewinnt ${.lte:7:${.random:10}:"${.locale:"SPORT_TEAMREPORT_MATCHKIND":"":1}":""} gegen
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_P1 = und Stürmer ${.self:"sportteam":0:"member":"forward":"fullname"} gewinnen ${.lte:7:${.random:10}:"${.locale:"SPORT_TEAMREPORT_MATCHKIND":"":1}":""} gegen
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_S2 = schlägt dank verwandelter Ecke durch ${.self:"sportteam":0:"member":-1:"lastname"} ${.lte:7:${.random:10}:"${.locale:"SPORT_TEAMREPORT_MATCHKIND":"":1}":""}
```

Comment:
Es sind noch ein paar "member":-1 und "member":0 übrig geblieben. Die sollten der Konsistenz wegen vielleicht noch in forward und keeper geändert werden.

Reply:
Ersetzt (dachte das haette ich schon .. hmpf)